packages/slider/src/spectrum-slider.css

```diff
@@ -408,278 +339,207 @@ governing permissions and limitations under the License.
-    left: auto; /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type */
+    left: auto;
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type {
-    margin-left: var(
-        --spectrum-slider-track-margin-offset
-    ); /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type */
+    margin-left: var(--spectrum-slider-track-margin-offset);
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type {
-    margin-right: var(
-        --spectrum-slider-track-margin-offset
-    ); /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type */
+    margin-right: var(--spectrum-slider-track-margin-offset);
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
     border-top-left-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
     border-top-right-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
     border-bottom-left-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
     border-bottom-right-radius: var(
         --spectrum-slider-m-track-border-radius,
         var(--spectrum-global-dimension-static-size-10)
-    ); /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    );
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
-    border-top-right-radius: 0; /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-top-right-radius: 0;
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
-    border-top-left-radius: 0; /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-top-left-radius: 0;
 }
 :host([dir='ltr'][variant='range']) .track:first-of-type:before {
-    border-bottom-right-radius: 0; /* [dir=ltr] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
+    border-bottom-right-radius: 0;
 }
 :host([dir='rtl'][variant='range']) .track:first-of-type:before {
-    border-bottom-left-radius: 0; /* [dir=rtl] .spectrum-Slider--range .spectrum-Slider-track:first-of-type:before */
-}
-:host([dir='ltr'][variant='range']) [dir='ltr'] .track,
-:host([dir='ltr'][variant='range']) [dir='rtl'] .track {
-    left: auto; /* [dir=ltr"] [dir="ltr] .spectrum-Slider--range .spectrum-Slider-track,
```

Comment:
Saw `[dir=ltr"] [dir="ltr]` and thought, "that's it, that's why," but the mismatched `"` are not in the original source... 😞 we may never know.
![theworldmayneverknow-idk](https://user-images.githubusercontent.com/1156657/226033020-7c873520-e7cd-4ced-8214-ac10352b55e2.gif)